### PR TITLE
finishing add modelnet

### PR DESCRIPTION
**Pull request by JerryJiehanWang**

### CODE REVIEW

File: tests/python/kaolin/io/test_modelnet.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+MODELNET_TEST_CATEGORY_LABELS_MULTI = ['bathtub', 'desk']
+
+ALL_CATEGORIES = [
+    MODELNET_TEST_CATEGORY_LABELS,
+    MODELNET_TEST_CATEGORY_LABELS_2,
+    MODELNET_TEST_CATEGORY_LABELS_MULTI,
+]
+
+# Skip test in a CI environment
+@pytest.mark.skipif(os.getenv('CI') == 'true' or os.getenv('CI'), reason="CI does not have dataset")
```
Comment (Caenorst) [inline tests/python/kaolin/io/test_modelnet.py]: I believe you just need `os.getenv('CI') == 'true'` now

File: kaolin/io/modelnet.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+class ModelNet(KaolinDataset):
+    r"""Dataset class for the ModelNet dataset.
+
+    Args:
+        root (str): Path to the base directory of the ModelNet dataset.
+        split (str): Split to load ('train' vs 'test', default: 'train').
+        categories (list): 
+            List of categories to load. If None is provided, 
+            all categories will be loaded. (default: None).
+    """
```
Comment (Caenorst) [inline kaolin/io/modelnet.py:34]: as in https://github.com/NVIDIAGameWorks/kaolin/pull/375#discussion_r617842222
We should add a little description of returned value from `__getitem__`

File: tests/python/kaolin/io/test_modelnet.py (hunk 1 above)
Comment (JerryJiehanWang) [reply tests/python/kaolin/io/test_modelnet.py]: fixed.

File: kaolin/io/modelnet.py (hunk 2 above)
Comment (JerryJiehanWang) [reply kaolin/io/modelnet.py:34]: fixed.

**Review by Caenorst — APPROVED** [review]
LGTM!

**Pull request merged**